index.bs

```diff
@@ -616,10 +630,77 @@ algorithm:
-header.
+For backwards compatibility, Suborigins provide several opt-opts from the
+standard security model. A developer can choose to use these opt-outs by
+specifying a <a>suborigin policy</a> in <a href="#the-suborigin-header">the
+suborigin header</a>
+
+Since these opt-outs weaken the security model of suborigins, developers SHOULD
+NOT use these options unless they are required to make their application work.
+
+The values of <a link-type="grammar">suborigin-policy-option</a> that may be
+present in a <a>suborigin policy</a> have the following effects:
+
+* '<dfn>`unsafe-postmessage-receive`</dfn>' When a message is sent <i>to</i> a
+  frame with a `postMessage` `target` of a serialized physical origin, but not a
+  serialized suborigin, if the frame has an execution context with a suborigin
+  where the scheme, host, and port match the `target`, but it also has a
+  suborigin namespace, if <a>`unsafe-postmessage-receive`</a> is set, it will
+  still receive the message.
+
+  <div class="example" id="unsafe-postmessage-send-ex">
+  `https://example.com` runs a map API at `https://example.com/maps` which is
+  embedded by many other websites. It provides a `postMessage()` API to place
+  markers on the map at locations the embedder chooses.
+
+  The developer would like to run `https://example.com/maps` in a suborigin
+  namespace "maps".  However, when embedders send messages to the embedded
+  frame, because they are legacy uses from before the use of suborigins, they
+  send essages with a `target` of `https://example.com`, <em>not</em>
+  `https://maps_example.com`. Since the developer would still like this frame to
+  be able to provide the API to these legacy embedders, it can serve the frame
+  with the [unsafe-postmessage-receive] directive, which will allow the
+  frame to receive messages on behalf of `https://example.com`, even though it is
+  at `https://maps_example.com`.
+  </div>
+
+* '<dfn>`unsafe-postmessage-send`</dfn>' When a message is sent <i>from</i> a suborigin
+  namespace with <a>`unsafe-postmessage-send`</a> set, the `event.origin` value
+  of the receiver should be set to the serialized physical origin, not the
+  serialized suborigin value. However, the `event.suborigin` field should still
+  be set to the name of the suborigin namespace.
+
+  <div class="example">
+  Continuing the case in the above example, `https://example.com/maps` is a
+  mapping application that is commonly embedded in other sites. It provides a
+  `postMessage()` based API to place locations of the embedder's choosing on the
+  map. `httsp://example.com` would like to run the application in a suborigin
+  named "maps".
+
+  In response to queries to the API, `https://example.com/maps` may send
+  messages back to the embedder if, for example, a user clicks on one of the
+  locations. However, since the embedder may be legacy and not be aware of
+  suborigins, when it checks the sender of the message, if it seem
```

Comment:
Sounds good, although I'm changing it to "...the event.origin property of the `MessageEvent`..."